source/rafcon/gui/utils/constants.py

```diff
@@ -28,6 +29,10 @@ def get_glade_path(glade_file):
 
 
 INTERFACE_FONT = "Source Sans Pro"
+if is_custom_design_enabled():
+    # this statement has no effect if the design_config is loaded after this module is imported the first time
+    INTERFACE_FONT = global_design_config.get_config_value("PRIMARY_FONT")
```

Comment:
Shouldn't all this be possible in one line of code?

`INTERFACE_FONT = global_design_config.get_config_value("PRIMARY_FONT", "Source Sans Pro")`